deployment/ansible/roles/cac-tripplanner.app/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+- name: Enable PostGIS
+  command: "psql -d {{ postgres_db }} -c 'CREATE EXTENSION postgis;'"
+  sudo_user: postgres
```

Comment:
PostGIS is enabled like this in NYC to help Ansible know the task is idempotent:

```yaml
- name: Add PostGIS extension
  sudo_user: postgres
  command: psql {{ postgresql_database }} -c "CREATE EXTENSION postgis"
  register: psql_result
  failed_when: >
    psql_result.rc != 0 and ("already exists" not in psql_result.stderr)
  changed_when: "psql_result.rc == 0"
```